main/src/main/scala/sbt/ClassLoaderLayeringStrategy.scala

```diff
@@ -89,36 +89,12 @@ object ClassLoaderLayeringStrategy {
   case object ScalaLibrary extends ScalaLibrary
 
   /**
-   * Add a layer on top of the ScalaLibrary layer for the runtime jar dependencies.
+   * Add a layer on top of the ScalaLibrary layer for all of the task jar dependencies.
    */
-  sealed trait RuntimeDependencies extends ScalaLibrary
+  sealed trait Dependencies extends ScalaLibrary
```

Comment:
The term "dependencies" comes up too often in the context of sbt, including `import Dependencies._`, so I think `RuntimeDependencies` is a bit more clearer than `Dependencies` even if it means sometimes test dependencies too.

Reply:
I'm open to a completely different name then.

Reply:
Kinda ugly, but maybe ScalaLibraryThenDependencies?

Reply:
`LibraryJars`, `ExternalJars`, `LayerDependencyJars`?

Reply:
I think of those I like ExternalJars the best just because ScalaLibrary already has Library in it.

Reply:
I picked `AllLibraryJars` to distinguish from `ScalaLibrary`.